java/sage/Seeker.java

```diff
@@ -1241,6 +1241,17 @@ else if (autoImportAddedNetPaths.size() > 0)
       writeArchiveDirsProps();
     }
 
+    // Even though this is the only method that manipulates this variable, it is run on a thread
```

Comment:
This is only for EMBEDDED code anyways, and there's no racing between threads here...while a different thread may do the library importing each time, there's loads of time between those executions so you'll never be in such a tight multi-threaded timing situation where volatile is needed.  It doesn't break anything for sure, and won't ever be executed, so I'm fine with you leaving it there or removing it. :)